sites/svelte.dev/scripts/type-gen/dts-sources/transition.d.ts

```diff
@@ -0,0 +1 @@
+export type * from 'svelte/transition';
```

Comment:
shouldn't these be part of .gitignore?

Reply:
If someone else clones the project and build, it won't work for them, so we should leave them in. In future, maybe we'd wanna monkey patch a specific type just for the docs, this would be a place for it